test/persistence-hooks.suite.js

```diff
@@ -99,6 +99,18 @@ module.exports = function(dataSource, should, connectorCapabilities) {
           });
       });
 
+      it('should trigger access hook once, if near filter is used', function(done) {
+        monitorHookExecution();
+        TestModel.find(
+          {where:
+            {location: {near: '10,20', maxDistance: '10', unit: 'meters'}},
+          }, function(err, list) {
```

Comment:
one arg per line please, see https://github.com/strongloop/loopback-contributor-docs/blob/198d4ebeb7171997474c05d6cc067adb60af3848/style-guide.md#one-arg-per-line

Alternatively, store the long query in a variable.


```js
var query = {
  where: {location: {near: '10,20', maxDistance: '10', unit: 'meters'}},
};

TestModel.find(query, function(err, list) {
  // ...
});
```